src/main/java/com/minecolonies/commands/ColoniesCommand.java

```diff
@@ -11,23 +16,44 @@
  */
 public class ColoniesCommand extends AbstractSplitCommand
 {
+    public static final String desc = "colonies";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00115)